src/server/plugins/engine/views/partials/form.html

```diff
@@ -1,7 +1,7 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList -%}
 
-<form method="post" enctype="multipart/form-data" autocomplete="on">
+<form method="post" enctype="multipart/form-data" autocomplete="on" novalidate>
```

Comment:
Can we remove the default `autocomplete="on"`?

Not sure if affects anything but we should set [per-field `autocomplete` attributes](https://design-system.service.gov.uk/components/text-input/#use-the-autocomplete-attribute) instead

```suggestion
<form method="post" enctype="multipart/form-data" novalidate>
```

Reply:
Resolved with 4526d79ca3ba089732f7db668abb36d8bff5e2a2